src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/UpdateSitesManager.java

```diff
@@ -116,7 +116,7 @@ public String getUrlName() {
      */
     public List<UpdateSite> getManagedUpdateSiteList() {
         return newArrayList(Iterables.filter(
-                Jenkins.getInstance().getUpdateCenter().getSites(),
+                Jenkins.getActiveInstance().getUpdateCenter().getSites(),

```

Comment:
<BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE' priority='Normal' category='STYLE' message='Possible null pointer dereference in jp.ikedam.jenkins.plugins.updatesitesmanager.UpdateSitesManager.getManagedUpdateSiteList() due to return value of called method' lineNumber='118'/>